src/it/scala/temple/containers/GolangSpec.scala

```diff
@@ -22,6 +22,7 @@ abstract class GolangSpec extends DockerShell2HttpService(8081) with DockerTestK
     val json = files.map { case (file, contents) => (file.folder + "/" + file.filename, contents) }.asJson.toString()
     Http(golangVerifyUrl)
       .params(Map("src" -> json, "root" -> entryFile.folder, "entrypoint" -> entryFile.filename))
+      .timeout(connTimeoutMs = 1000, readTimeoutMs = 10000)
```

Comment:
@xsanda - this should have fixed the error we were seeing with Travis last night. The time taken to compile the Go exceeded the read timeout of the HTTP request, so I've bumped the default read value to 10000ms